rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -177,7 +300,15 @@ components:
         dev:
           value: '1.2.0-SNAPSHOT'
           summary: A development version
-
+    Location:
+      description: >
+        The URI that can be used to poll for readiness of the transformed file's discovery metadata and other supporting products.
```

Comment:
✏️ How about just saying that this is the URI from which to retrieve the status of the asynchronous transformation request.